packages/compass-web/sandbox/index.tsx

```diff
@@ -144,7 +144,7 @@ const App = () => {
                 isAtlas && !!enableGenAIFeaturesAtlasOrg,
               optInDataExplorerGenAIFeatures:
                 isAtlas && !!optInDataExplorerGenAIFeatures,
-              enableDataModeling: false,
+              enableDataModeling: true,
```

Comment:
@Anemy asked before to keep it switched off to make sure it's aligned with desktop